demo/src/components/RangeView.js

```diff
@@ -0,0 +1,287 @@
+import React from 'react';
+import RangeSlider from './RangeSlider';
+import { RangeFacets } from '@unbxd-ui/react-search-sdk';
+import {executeCallback} from './RangeFacetsUtils';
+import { getUpdatedRangeFacets } from './RangeFacetsUtils';
+import { scrollTop } from '../utils';
+
+const transform = function () {
```

Comment:
coz the reason for providing the hook still applies

Reply:
Do we still need to pass the transform as a prop though? The override component will receive the range facets data and it can be transformed by the integrator.

Reply:
yes you are right, the integrator can transform the range facets after receiving it but in that case there would be a difference in the internal state of the sdk and the what the integrator would be integrating